src/components/Category/Category.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "./category.css";
+
+const Category = props => (
```

Comment:
You can use `const Category = ({ name }) => (` and then on line 7 you can have `{name}`. It is ES6 object deconstruction - it really helps when you need to work with more variables.

Reply:
nifty trick!